cirq-core/cirq/sim/act_on_density_matrix_args.py

```diff
@@ -36,8 +36,8 @@ class ActOnDensityMatrixArgs(ActOnArgs):
     def __init__(
         self,
         target_tensor: np.ndarray,
-        available_buffer: List[np.ndarray],
-        qid_shape: Tuple[int, ...],
+        available_buffer: List[np.ndarray] = None,
+        qid_shape: Tuple[int, ...] = None,
```

Comment:
These need their types changes to `Optional[...]`. (Not sure why mypy didn't catch this...)

Reply:
I was curious about this too. It's explained in the PEP. It works implicitly for now but is no longer recommended, and tooling will soon be changed to catch these. https://www.python.org/dev/peps/pep-0484/#union-types

Reply:
Sorry, I will change them...